README.md

```diff
@@ -32,24 +38,48 @@ var keyContent = fs.readFileSync(keyFile);
 
 var view = new DataView(keyContent.buffer);
 var key.bytes = new Uint32Array([
-	view.getUint32(0),
-	view.getUint32(4),
-	view.getUint32(8),
-	view.getUint32(12)
+  view.getUint32(0),
+  view.getUint32(4),
+  view.getUint32(8),
+  view.getUint32(12)
 ]);
 
 key.iv = new Uint32Array([
-	0, 0, 0, 0
+  0, 0, 0, 0
 ]);
 
 var d = new Decrypter(
-	encryptedBytes,
-	key.bytes,
-	key.iv,
-	function(err, decryptedBytes) {
-		// err always null
+  encryptedBytes,
+  key.bytes,
+  key.iv,
+  function(err, decryptedBytes) {
+    // err always null
 });
+```
+
+## Command Line Usage
```

Comment:
Probably shouldn't have anything regarding CLI usage if the bin file is not being included